dags/mantaray/run_mantaray_jobs.py

```diff
@@ -41,7 +41,7 @@
   # merge all PyTorch/XLA tests ino one Dag
   with models.DAG(
       dag_id="pytorch_xla_model_regression_test_on_trillium",
-      schedule="0 0 * * *",  # everyday at midnight # job["schedule"],
+      schedule="0 0 * * *" if composer_env.is_prod_env() else None,  # everyday at midnight # job["schedule"],
```

Comment:
hi, @RissyRan, due to other test might been set with different schedule time, so I only update test for `pytorch_xla_model_regression_test_on_trillium`, and please let me know if I missed anything

Reply:
Could you upload to an test env, and see if other child dags get benefit? I think it should work properly.